modules/ROOT/pages/policies-policy-overview.adoc

```diff
@@ -0,0 +1,148 @@
+endif::[]
+:keywords: policy, custom, ootb, offline
+//where are we covering what a policy? What the policy does
+
+Policies differ based on several different factors, such as level of granularity,types of policies, and policy categories. You must carefully review these factors, before you decide to implementation them in your environment. In this section, each of these factors are explained in detail. 
+
+// == Policy Components
+
+// A policy consists of the following components:
+
+// * YAML file
+// +
+// Defines the configurable parameters of the policy. Rendered by the UI to display the input of the policy.
+// +
+// * XML template
+// +
+// Contains the implementation of the policy.
+// * Template's POM
+// // Is it clear in the navigation that this is only for Mule 4? Mule 3 policies don't have a POM or a descriptor. -- Eva?
+// // What is POM?
+// +
+// Defines policy dependencies. Packaging type needs to be `mule-policy`.
+// * Resources
+// +
+// Optionally if the policy depends on other resources, such as a certificate, define resources as part of the policy.
+// +
+// * Mule-artifact.json descriptor
+// +
+// Specify the classifier as `mule-policy`.
+
+// For more information, see the header policy.
+
+== Policy Scope
+
+Policies are isolated from the application and other policies. However, there are ways to expose information from a policy. The authentication information about a policy can be propagated in the Security Context Principal object.
+
+//Eva--add info about the generic policy scope Add an example or link here..Fede Balbi?
+
+// A policy can modify message content. In Mule 4.1.0 and later, if the message is modified before the `execute-next` in `source` policies
+// or after it in `operation` policies, those modifications are not propagated by default. To enable propagation of those modifications
+// you can enable the `propagateMessageTransformations` flag in each policy.
+
+// *Example*
+
+// ----
+// <http-policy:proxy name="scope-payload">
+//     <http-policy:source propagateMessageTransformations="true">
+
+//         <http-policy:execute-next/>
+
+//     </http-policy:source>
+//     <http-policy:operation propagateMessageTransformations="true">
+
+//         <http-policy:execute-next/>
+
+//     </http-policy:operation>
+// </http-policy:proxy>
+// ----
+
+// A policy can avoid the execution of the flow by not defining the `execute-next` or making sure that it is not reached (placing it in a choice, for example).
+
+// Variables set on a policy have only the scope of the policy. Also, variables defined in the flow, are not visible in policies.
+
+== Policy Size
+
+The size of a policy varies based on its dependencies. The first time Mule Runtime polls for policies it will take longer to fetch them. However, after policies are retrieved, they are cached on the file system. This ensures offline availability and also reduces latency to fetch policies.
+//Add more information about the policy size.
```

Comment:
@chiragnarangmulesoft we have two options here, either hide the policy size, or clrify that this doesn't happen for cloudhub or runtime fabric

Reply:
Add a comment to clarify with Chirag